packaging/leapp-repository.spec

```diff
@@ -26,11 +23,29 @@ Repositories for leapp
 %package data
 License: Red Hat Enterprise Agreement
 Summary: Package evolution data for leapp
-Requires: %{name} = %{version}-%{release}
+Requires:   %{name} = %{version}-%{release}
 
 %description data
 Package evolution data for leapp.
 
+%package deps
+Summary:    Meta-package with system dependencies of %{name} package
```

Comment:
I think it would be worth to add a comment here about why do we create this meta package.